### ISSUE TITLE
On CTR 1.4.7, toolbar button badges are now too small and size can't be overridden?

### ISSUE THREAD

**Issue by camartinez1229** [Aris-t2/ClassicThemeRestorer#159]
Hello. I use the Cyberfox browser, which uses a customized version of CTR. I updated the browser from 44.0 to 44.0.1, which also updated its CyberCTR (customized version of CTR) to incorporate changes from CTR 1.4.7. Now my toolbar button notification badges are too small, and I can't seem to override that with Stylish or userChrome.css?

I tracked down the offending change to line 403 in winaero/smallnavbutextra.css, the `"font-size:60% !important`" bit. That part was unchanged from the original addon, so, I don't think it's anything to do with any Cyberfox customizations.

Anyway, I try plugging "`#navigator-toolbox #nav-bar toolbarbutton .toolbarbutton-badge-stack .toolbarbutton-badge`" into Stylish and play around with the font-size, but, nothing works? Oddly, I can set other properties like background-color if I want, but I can't override the font-size. So, I tried to change it using userChrome.css as well, and still nothing. If I want to change it, I have to edit smallnavbutextra.css, which is not ideal, because I'm sure my change will get overwritten on the next browser update.

I'm using Cyberfox on Windows 7 with a 1920x1080 resolution. I find 80% a more legible font-size. But what I want to know is, is it possible to make that user-configurable without having to edit the addon CSS directly? I don't understand why I couldn't override the font-size with Stylish or userChrome.css, especially when I could set other properties like font-family or background-color.

**Comment by Aris-t2** [comment]
UserChrome to override most of CTRs code, because userChrome is way too weak.

Use Stylish add-on and ```/*AGENT_SHEET*/``` in first line of a style.

```css
/*AGENT_SHEET*/
#main-window #navigator-toolbox #nav-bar toolbarbutton .toolbarbutton-badge-stack .toolbarbutton-badge {
  font-size:75% !important;
}
```

I will increase the size to 75% in CTR 1.4.8

**Comment by camartinez1229** [comment]
Awesome! That works for me in Stylish. Thank you! I had no idea about the `/*AGENT_SHEET*/` bit before, but I guess that's what makes it work.

**Comment by Aris-t2** [comment]
Fixed in CTR 1.4.8
